support-e2e/tests/utils/users.ts

```diff
@@ -25,5 +23,4 @@ export const lastName = () =>
  *
  * @see https://github.com/guardian/membership-workflow/blob/99e2b90305f93bf35ce230f6b6c17e0c4533facb/app/model/BrazeCampaignTriggerPayload.scala#L28
  **/
-export const email = () =>
-  `test.support.frontend.e2e+${uuidv4()}@thegulocal.com`;
+export const email = () => `e2e.email+${uuidv4()}@thegulocal.com`;
```

Comment:
The reason support-frontend is there is so we know where these tests have come from (using a new engineer as a use case).

Maybe we could just use something like https://github.com/ai/nanoid/?

Reply:
Ok cool, yeah nice suggestion, will look at that now

Reply:
I've updated the tests to use the `nanoid` library instead of `uuid`. One thing to note is I'm using `nanoid@v3` instead of  the latest `nanoid@v5`. I initially went to `nanoid@v5` but ran into an issue as `nanoid@v5` works only with ESM projects, in tests or Node.js scripts - and Playwright has issues working with ESM modules, this issue in Github neatly summarises the issues I was running into with PlayWright and ESM: https://github.com/microsoft/playwright/issues/23662

For CommonJS you need `nanoid@v3` (this is outlined in Nano's [docs](https://github.com/ai/nanoid?tab=readme-ov-file#install)), so I've switched to this and it works as desired.

Reply:
I won't suggest to change it now but I'm sure time in milliseconds since epoch would have been good enough and I'm sure you don't need a library.
Having a rough idea of when it was created is probably not important but it would be a side effect too.